service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterMapper.java

```diff
@@ -53,10 +57,36 @@ public String mapEncounterToEhrComposition(Encounter encounter) {
             .displayName(buildDisplayName(encounter))
             .originalText(buildOriginalText(encounter));
 
+        final Optional<String> recReference = findParticipantWithCoding(encounter, "REC")
+            .map(idMapper::get);
+
+        recReference.ifPresent(encounterStatementTemplateParameters::author);
```

Comment:
Make sure not to override the change in https://github.com/nhsconnect/integration-adaptor-gp2gp/pull/156/files to make this mandatory.